torch/testing/_internal/common_methods_invocations.py

```diff
@@ -131,6 +131,8 @@ def __call__(self, *args, **kwargs):
 
 _NOTHING = object()  # Unique value to distinguish default from anything else
 
+_DYNAMIC_DTYPES = _dispatch_dtypes(tuple())
```

Comment:
Is this distinguishable from supporting no dtypes on a device?

Reply:
Yes. Unique instance of class `_dispatch_dtypes`

https://github.com/pytorch/pytorch/blob/74c12da4517c789bea737dc947d6adc755f63176/torch/testing/_core.py#L311